src/app/Fake.Core.Trace/TraceListener.fs

```diff
@@ -183,6 +183,14 @@ module TraceData =
         | TraceData.TraceMessage (text, d) -> TraceData.TraceMessage (f text, d)
         | TraceData.TestStatus (testName,status) -> TraceData.TestStatus(testName, TestStatus.mapMessage f status)
         | TraceData.TestOutput (testName,out,err) -> TraceData.TestOutput (testName,f out,f err)
+        | TraceData.OpenTag(KnownTags.Task tag, Some d) -> TraceData.OpenTag(KnownTags.Task (f tag), Some(f d))
+        | TraceData.OpenTag(KnownTags.Target tag, Some d) -> TraceData.OpenTag(KnownTags.Target (f tag), Some(f d))
+        | TraceData.OpenTag(KnownTags.FinalTarget tag, Some d) -> TraceData.OpenTag(KnownTags.FinalTarget (f tag), Some(f d))
+        | TraceData.OpenTag(KnownTags.FailureTarget tag, Some d) -> TraceData.OpenTag(KnownTags.FailureTarget (f tag), Some(f d))
```

Comment:
You can combine those with a `mapKnownTagsMessage` function (which you could re-use for `CloseTag` as well)